website/docs/d/batch_pool.html.markdown

```diff
@@ -92,3 +94,15 @@ A `auto_user` block exports the following:
 * `elevation_level` - The elevation level of the user identity under which the start task runs.
 
 * `scope` - The scope of the user identity under which the start task runs.
+
+---
+
+A `certificate` block exports the following:
+
+* `id` - The fully qualified ID of the certificate installed on the pool.
+
+* `store_location` - The location of the certificate store on the compute node into which the certificate is installed. Possible values are `CurrentUser` or `LocalMachine`. The default value is `CurrentUser`. This property is applicable only for pools configured with Windows nodes (that is, created with cloudServiceConfiguration, or with virtualMachineConfiguration using a Windows image reference). For Linux compute nodes, the certificates are stored in a directory inside the task working directory and an environment variable AZ_BATCH_CERTIFICATES_DIR is supplied to the task to query for this location. For certificates with visibility of 'remoteUser', a 'certs' directory is created in the user's home directory (e.g., /home/{user-name}/certs) and certificates are placed in that directory.
```

Comment:
since this is a Data Source I think this'd be more consistent as:

```suggestion
* `store_location` - The location of the certificate store on the compute node into which the certificate is installed, either `CurrentUser` or `LocalMachine`.

-> **NOTE:** This property is applicable only for pools configured with Windows nodes (that is, created with cloudServiceConfiguration, or with virtualMachineConfiguration using a Windows image reference). For Linux compute nodes, the certificates are stored in a directory inside the task working directory and an environment variable `AZ_BATCH_CERTIFICATES_DIR` is supplied to the task to query for this location. For certificates with visibility of 'remoteUser', a 'certs' directory is created in the user's home directory (e.g., `/home/{user-name}/certs`) and certificates are placed in that directory.
```